config.py

```diff
@@ -123,7 +123,7 @@ def get_url_endpoint(default_url, endpoint_type=None):
         endpoint_type
     )
     if endpoint_type == 'flare':
-        subdomain_replacement = "{0}.flare".format(formatted_version)
+        subdomain_replacement = "{0}-flare".format(formatted_version)
 
```

Comment:
Not sure how to say it: `*.flare.datadoghq.com` would be a new subdomain, `1-2-3-flare.datadoghq.com` is part of `*.datadoghq.com`.